src/controller/raid_floor.controller.ts

```diff
@@ -0,0 +1,115 @@
+import { UpdatedAt } from 'sequelize-typescript';
+import { Floor } from '../models/floor.model.js'
+
+
+///////Read or Pull all
+export const findAllFloor = async (req, res) => {
+    try {
+        console.log("Calling find all for Loot on the Table")
+        const allFloor:Floor[] = await Floor.findAll();
+        res.status(200).json({message: allFloor})
+    } catch (error) {
+       res.status(500).json({ message: error.message });
+    }
+}
+
+//Pull a SINGLE PERSON NEEDS TO BE FIXED
+// export const findSingleLootTable = async (req, res) => {
+//     try {
+//         console.log("Calling find all for Static Mates")
+// //         const allLootTables:Floor[] = await Floor.findAll();
+// //         res.status(200).json({message: allLootTables})
+//     } catch (error) {
+//         res.status(500).json({ message: error.message });
+//     }
+// }
+
+///create
+
+export const createSingleFloor = async (req, res) => {
+    try {
+        //properties on the query object and the query object is property on the request object 
+        const { floor_acronym, floor_full_name } = req.body;
+        console.log("req.body", req.body)
+        //find a single pesron
+        const findSingleFloor = await Floor.findOne({ where: {floor_abbreviation: floor_acronym} });
```

Comment:
Type needed

Reply:
Same as mentioned above, I think I did it right but still concerned about being not 100% confident